### PR TITLE
feat: extract tag number framework

### PR DESCRIPTION
**Pull request by herda1**
## Type(s) of Changes
- Move tagNumber component from EchopediaWeb to Framework 

Related PRs: 
- https://github.com/equinor/EchopediaWeb/pull/1063
### Contribution Type

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

-   [ ] Bug fix (non-breaking change which fixes an issue, please reference the issue id)
-   [x] New feature (non-breaking change which adds functionality, make sure to open an associated issue first)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)
-   [ ] I have updated the documentation accordingly
-   [ ] I have added tests to cover my changes

### Remarks

Have to update the Framework version for the changes in EchopediaWeb to occur.

### CODE REVIEW

**Review by OveBDev — APPROVED** [review]

File: packages/EchoFramework/src/components/tagNumber/tagNo.module.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.tagNo{
```
Comment (CsabaTheKiss) [inline packages/EchoFramework/src/components/tagNumber/tagNo.module.css:1]: Missing space before bracket.

File: packages/EchoFramework/src/components/tagNumber/tagNo.module.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+    cursor: pointer !important;
```
Comment (CsabaTheKiss) [inline packages/EchoFramework/src/components/tagNumber/tagNo.module.css:2]: Is this `!important` still needed? As I see this is applied to just a div, not a pre-made component.

Comment (herda1) [reply packages/EchoFramework/src/components/tagNumber/tagNo.module.css]: true, it wasnt needed

**Comment by CsabaTheKiss** [comment]
Just a remark: from echoFramework perspective this isn't a breaking change. All changes in this PR are new additions.

**Review by CsabaTheKiss — APPROVED** [review]

**Pull request merged**